pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -423,3 +415,26 @@ func getCRICadvisorStats(ca cadvisor.Interface) (map[string]cadvisorapiv2.Contai
 	}
 	return stats, nil
 }
+
+func getContainerLogStats(path string) *statsapi.FsStats {
+	m := volume.NewMetricsDu(path)
```

Comment:
`NewMetricsDu` returns an interface `MetricsProvider`. It should be easy to mock it.
Let's use a mock instead of using actual log stats.

Reply:
done